pandas/tests/series/test_analytics.py

```diff
@@ -789,6 +789,38 @@ def test_corr_invalid_method(self):
         with tm.assert_raises_regex(ValueError, msg):
             s1.corr(s2, method="____")
 
+    def test_corr_callable_method(self):
+        # simple correlation example
+        # returns 1 if exact equality, 0 otherwise
+        my_corr = lambda a, b: 1. if (a == b).all() else 0.
+
```

Comment:
can you use result= and expected= here, rather than expected_1 and such. its much easier to follow

Reply:
done